docs/source/config.json

```diff
@@ -8,17 +8,23 @@
     "Quickstart": "/quickstart",
     "Moving from @apollo/gateway": "/migrating-from-gateway",
     "Federation version support": "/federation-version-support",
-    "Configuration": {
+    "Configuring the Router": {
       "Overview": "/configuration/overview",
-      "JWT Authentication": "/configuration/authn-jwt",
       "Caching": "/configuration/caching",
-      "CORS": "/configuration/cors",
-      "CSRF prevention": "/configuration/csrf",
       "External extensibility": "/configuration/external",
-      "Logging": "/configuration/logging",
-      "Header propagation": "/configuration/header-propagation",
-      "Traffic shaping": "/configuration/traffic-shaping",
-      "Subgraph error inclusion": "/configuration/subgraph-error-inclusion"
+      "Debugging": {
```

Comment:
@abernix Do we support this extra layer of nesting?

Reply:
I'm assuming the answer is yes, but I'm only assuming that the answer is yes because one of the people I would check with is the author of this PR (whose team owns the documentation framework).

Reply:
We do! We don't use it in a ton of places, but this category's getting large enough that it feels helpful